pom.xml

```diff
@@ -354,39 +323,40 @@
   </reporting>
 
   <profiles>
-
     <profile>
-      <id>publish</id>
-
-      <build>
-        <plugins>
-          <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-source-plugin</artifactId>
-            <executions>
-              <execution>
-                <id>attach-sources</id>
-                <goals>
-                  <goal>jar</goal>
-                </goals>
-              </execution>
-            </executions>
-          </plugin>
-
-          <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-javadoc-plugin</artifactId>
-            <executions>
-              <execution>
-                <id>attach-javadocs</id>
-                <goals>
-                  <goal>jar</goal>
-                </goals>
-              </execution>
-            </executions>
-          </plugin>
-        </plugins>
-      </build>
+      <id>default</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
I would be more interested in making sure it works now and keep an eye on the main pom. Again, it is very important to have a profile that disables everything that is not related to the OpenShift deliverables. And the task is even more complicated now because the two JAX-RS implementations; the project will be simplified in the next months and we can revisit this at that time.